icu4c/source/data/buildtool/__main__.py

```diff
@@ -122,6 +128,9 @@ def __init__(self, args):
         # Either "unihan" or "implicithan"
         self.coll_han_type = args.collation_ucadata
 
+        # Boolean: whether to include unicore data in dat file
```

Comment:
Whether to include core Unicode data files in the .dat file.

Reply:
Done